ancestrymanager/ancestryutil.go

```diff
@@ -91,10 +95,14 @@ func getOrganizationFromResource(tfData resources.TerraformResourceData) (string
 	if ok {
 		return orgID.(string), ok
 	}
+	orgID, ok = tfData.GetOk("parent")
+	if ok && strings.HasPrefix(orgID.(string), "organizations/") {
+		return orgID.(string), ok
+	}
 	return "", false
 }
 
-// getFolderFromResource reads folder_id or folder field from terraform data.
+// getFolderFromResource reads folder_id, folder, parent field from terraform data.
 func getFolderFromResource(tfData resources.TerraformResourceData) (string, bool) {
 	folderID, ok := tfData.GetOk("folder_id")
```

Comment:
For a folder that has already been created, this function will currently return the value of `folder_id`, which is a string of numbers `XXXX` - but it needs to instead return `folders/XXXX`. However, for most resources, `folder_id` will _already_ be in the format `folders/XXXX`. So, it needs to conditionally add the prefix if it's not already present.

Reply:
The test for this could be something like `Google folder with organizations/ as {parent} and folder_id set`